arm/Microsoft.Management/managementGroups/deploy.bicep

```diff
@@ -311,12 +27,11 @@ resource managementGroup 'Microsoft.Management/managementGroups@2021-04-01' = {
 
 module managementGroup_rbac '.bicep/nested_rbac.bicep' = [for (roleAssignment, index) in roleAssignments: {
   name: '${deployment().name}-rbac-${index}'
-  scope: az.managementGroup(managementGroup.name)
   params: {
     managementGroupName: managementGroup.name
-    builtInRoleNames: builtInRoleNames
     roleAssignmentObj: roleAssignment
   }
+  scope: az.managementGroup(managementGroup.name)
```

Comment:
az?

Reply:
Simplified the scoping to just be `scope: managementGroup` as this is the symbolic name/reference to the managementGroup which is created/maintained.